test/hotspot/jtreg/vmTestbase/nsk/jdi/VMOutOfMemoryException/VMOutOfMemoryException001/VMOutOfMemoryException001.java

```diff
@@ -79,7 +79,11 @@ public void doTest() {
             // create array in debuggee VM till VMOutOfMemoryException
             while (true) {
                 ArrayReference array = referenceType.newInstance(100000);
-                array.disableCollection();
+                try {
+                    array.disableCollection();
+                } catch (ObjectCollectedException e) {
+                    continue;
```

Comment:
Maybe add a comment: "Since the VM is not suspended, the object may have been collected before disableCollection() could be called on it. Just ignore and continue doing allocations until we run out of memory."

Reply:
Sounds good, will fix.